core/services/directrequest/delegate.go

```diff
@@ -190,16 +190,15 @@ func (l *listener) run() {
 		case <-l.chStop:
 			l.shutdownWaitGroup.Done()
 			return
-		case head := <-l.chHeads:
-			l.handleReceivedLogs(head)
+		case <-l.chHeads:
```

Comment:
Hmm, I'm curious now why the direct request delegate needs to be head trackable at all... can we not simply subscribe to the log broadcaster and forget about heads?

Reply:
true, yes - we actually made it head trackable only recently, to have custom confirmation logic